carberretta/bot/cogs/youtube.py

```diff
@@ -10,6 +10,7 @@
 import aiohttp
 import discord
 from discord.ext import commands
+import html
```

Comment:
Sort imports as per PEP 8 standards.
```suggestion
import html
from discord.ext import commands
```